AUTHORS

```diff
@@ -1,10 +0,0 @@
-Sean DuBois <https://github.com/Sean-Der>
-Raphael Randschau <https://github.com/nicolai86>
-Aleksandr Razumov <ar@cydev.ru>
-Aliaksandr Valialkin <valyala@gmail.com>
-Michiel De Backker <https://github.com/backkem>
-Y.Horie <https://github.com/u5surf>
-songjiayang <https://github.com/songjiayang>
-The gortc project
-The IETF Trust
```

Comment:
I heard this repository was originally from gortc organization.

Reply:
What do you think about adding them directly to the Contributing section of the README?

I think the generate-authors script from pion/goassets is not flexible enough to handle this case..

Reply:
I added them to the README